metacat-main/src/main/java/com/netflix/metacat/main/services/notifications/sns/SNSNotificationServiceUtil.java

```diff
@@ -174,6 +179,25 @@ public TablePartitionsUpdatePayload createTablePartitionsUpdatePayload(
             .collect(Collectors.toList());
     }
 
+    /**
+     * get partition name list from list of partitionDtos. The returned list is capped at
+     * the first PARTS_UPDATED_LIST_MAX_SIZE elements, if there are more than that number of elements
```

Comment:
should we just call this PARTITIONS_UPDATED_LIST_MAX_SIZE

Reply:
ok changed